packages/manager/apps/pci-billing/src/api/hook/useConsumption.tsx

```diff
@@ -147,6 +147,35 @@ const initSnapshotList = (data: TCurrentUsage) => {
   };
 };
 
+const initRancherList = (data: TCurrentUsage) => {
+  if (!data.hourlyUsage.rancher) {
```

Comment:
`rancher` is always defined in its type, so what do you want to achieve with this assertion? 

If you want to check length you should do so:

```suggestion
  if (!data.hourlyUsage.rancher.length) {
```